projects/mercury/src/file/FileBrowser.js

```diff
@@ -107,42 +107,40 @@ export const FileBrowser = (props: FileBrowserProperties) => {
     const [overwriteFileCandidateNames, setOverwriteFileCandidateNames] = useState([]);
     const [overwriteFolderCandidateNames, setOverwriteFolderCandidateNames] = useState([]);
     const [currentUpload, setCurrentUpload] = useState({});
-    const [isFolderUpload, setIsFolderUpload] = useState();
 
-    const {
-        getRootProps,
-        getInputProps,
-        isDragActive,
-        isDragAccept,
-        isDragReject,
-        open
-    } = useDropzone({
-        noClick: true,
-        noKeyboard: true,
-        multiple: true,
-        onDropAccepted: (droppedFiles) => {
-            const newUpload = {
-                id: generateUuid(),
-                files: droppedFiles,
-                destinationPath: openedPath,
-            };
-            const newOverwriteFolderCandidates = getConflictingFolders(droppedFiles, existingFolderNames);
-            const newOverwriteFileCandidates = getConflictingFiles(droppedFiles, existingFileNames);
+    function InitializeDropzone() {
```

Comment:
```suggestion
    const initializeDropzone = () => {
```
Just for consistency with the way we define functions in other places